src/main/scala/com/evojam/play/elastic4s/client/ElasticSearchClient.scala

```diff
@@ -112,6 +110,43 @@ trait ElasticSearchClient {
    */
   @throws[NotAJsObjectException[_]]("if some of the documents serialize to something that is not a Json object")
   def bulkInsert[T: Writes](doctype: IndexType, documents: Iterable[T]): Future[BulkResponse]
+
+  /**
+    * Prepares a get request for a document with a given ID.
+    *
+    * @param id id of the requested document
+    * @param docType index name and document type
+    * @param exc the execution context
+    * @return a [[PreparedGet]] instance encapsulating the query
+    */
+  def get(id: String, docType: IndexType)(implicit exc: ExecutionContext): PreparedGet
```

Comment:
`id: String`

Worries me a bit, since in `elastic4s` the type of `id` is `Any`. @WojciechP @abankowski Do you think we should use `Any` here for compatibility, or (preferably ;P ) any other ideas?